### PR TITLE
Support GitHub dark theme

### PR DESCRIPTION
**Pull request by hyrious**
This script did not follow the `uncss` way to cleanup css, instead it manually filtered out interested styles from GitHub.

To test if it work, run `npm test` or `node test.js` to generate an `index.html` and `github-markdown.css` in the dist folder.

Currently it only generates "one theme" file or "two theme" file. But it is possible to generate every combination of GitHub themes since we have gathered all information we need, thanks to GitHub for separating each theme in one file.

You can suggest what the output should look here. Here I'll paste two common cases:
- light-dark by media query (prefers-color-scheme) [light-dark.css.txt](https://github.com/sindresorhus/generate-github-markdown-css/files/7315018/light-dark.css.txt)
- dark only [dark.css.txt](https://github.com/sindresorhus/generate-github-markdown-css/files/7315019/dark.css.txt)

Related Issue: sindresorhus/github-markdown-css#76

### CODE REVIEW

**Comment by sindresorhus** [comment]
Hmm, actually, I think a lot of Markdown editors will want to have control over dark/light, so maybe we can provider 3 versions:

- Both
- Light
- Dark

?

File: cli.js [hunk 1]
```diff
@@ -3,13 +3,35 @@
...
 
-meow(`
+const cli = meow(`
 	Usage
 	  github-markdown-css > <filename>
+
+	Options
+	  --light     Light theme
+	  --dark      Dark theme
+	  --list, -l  List available themes
```
Comment (sindresorhus) [inline cli.js:13]: I don't see the point of a short flag for this. It's not a command people will run often.

File: readme.md [hunk 2]
```diff
@@ -30,4 +30,13 @@ $ github-markdown-css --help
...
     $ github-markdown-css > <filename>
+
+  Options
+    --light     Light theme
+    --dark      Dark theme
+    --list, -l  List available themes
+
+  Examples
+    $ github-markdown-css -l
+    light, dark, dark_dimmed, dark_high_contrast, dark_colorblind, light_colorblind
```
Comment (sindresorhus) [inline readme.md:41]: I would list each theme on a new line.

File: cli.js [hunk 3]
```diff
@@ -8,30 +8,39 @@ const cli = meow(`
...
 	Examples
-	  $ github-markdown-css -l
-	  light, dark, dark_dimmed, dark_high_contrast, dark_colorblind, light_colorblind
+	  $ github-markdown-css --list
+	  light
+	  dark
+	  dark_dimmed
+	  dark_high_contrast
+	  dark_colorblind
+	  light_colorblind
```
Comment (sindresorhus) [inline cli.js:22]: We should explain somewhere what `auto` implies.

Comment (hyrious) [reply cli.js:22]: I'll add that to the description of `--type`.

**Comment by sindresorhus** [comment]
Thank you so much for working on this. 🙌

**Pull request merged**